fs_gcsfs/_gcsfs.py

```diff
@@ -70,6 +74,9 @@ def __init__(self,
         self.client = client
         if self.client is None:
             self.client = Client()
+            if retry:
```

Comment:
`if retry is not None`

Reply:
Why should we enter the if case when `retry==0` or `retry==False` or `retry==""`?

Reply:
Yes, that's a good point.